topics/quickstart.md

```diff
@@ -3,76 +3,151 @@ title: "Quick start guide"
 description: Understand how to use basic Valkey data types
 ---
 
-This quick start guide shows you how to:
+# Getting Started with Valkey
```

Comment:
We don't use level 1 headings in theae files. The level 1 heading is created from the 'title' attribute.